src/api/exercises.js

```diff
@@ -2,21 +2,33 @@ import { Zeeguu_API } from "./classDef";
 import qs from "qs";
 
 Zeeguu_API.prototype.getUserBookmarksToStudy = function (count, callback) {
```

Comment:
Maybe rename to align with the endpoint? Otherwise the difference between `toStudy` and `toLearn` below is too subtle :)